packages/dds/sequence/src/test/sharedString.spec.ts

```diff
@@ -395,6 +395,26 @@ describe("SharedString", () => {
             ]);
         });
 
+        it.only("can slide intervals on create conflict with remove range", () => {
```

Comment:
Needs `.only` removed. (I hope fluid lint fails with this committed; I want to check that because if it doesn't we should adjust our lint config...)

Reply:
ah, beat me to it. 🥂

Reply:
I accepted Tony's fix before seeing this note.

Reply:
@anthony-murphy do you know if lint bans `.only` when run on CI? We took that approach in Whiteboard after an accidental checkin of a `.only` missed in review.

Reply:
er, actually I think we didn't use lint for this, instead passed `--forbid-only` (see [here](https://mochajs.org/#command-line-usage)) on CI test command. But yeah, *some* kind of validation is all I want